Examples/Framework/include/ActsExamples/EventData/ProtoTrack.hpp

```diff
@@ -12,10 +12,12 @@
 
 #include <vector>
 
+#include <boost/container/small_vector.hpp>
+
 namespace ActsExamples {
 
 /// A proto track is a collection of hits identified by their indices.
-using ProtoTrack = std::vector<Index>;
+using ProtoTrack = boost::container::small_vector<Index, 10>;
```

Comment:
this uses 3x more space than necessary for 3 SP, right?

Reply:
Yeah true, I just went with a low-ish number. Let's see what I get with 3.

Reply:
In my testing, reducing to 3 appears to actually slow things down by about 3%, which I can't explain. I suggest we keep it at 10.